### ISSUE TITLE
Missing upscale_on_initial_job on JobLatency resource

### ISSUE THREAD

**Issue by stevenharman** [carwow/terraform-provider-hirefire#33]
Thank you for building and maintaining this Terraform provider.

The `Manager::Worker::HireFire::JobLatency` manager type is missing a newer parameter, `upscale_on_initial_job`. This is a boolean value. 

~It's currently missing from the docs. I've asked HireFire to add it to the docs.~ It's now [documented in HireFire's docs](https://docs.hirefire.io/#manager-worker-hirefire-joblatency), and you can it in API responses if you fetch/list `managers` for an app. Here's an example API response:

```json
{
  "manager": {
    "id": "abc12345-aaaa-bbbb-cccc-000000000000",
    "type": "Manager::Worker::HireFire::JobLatency",
    "application_id": "12345678-zzzz-zzzz-zzzz-111111111111",
    "name": "within_5_minutes",
    "enabled": true,
    "metric_value": "20",
    "dyno_quantity": 1,
    "last_checkup_time": "2024-03-13T17:25:24.336Z",
    "last_scale_time": "2024-01-12T15:38:52.655Z",
    "minimum": 0,
    "maximum": 5,
    "notify": false,
    "notify_quantity": 0,
    "notify_after": 0,
    "created_at": "2023-07-10T20:56:58.643Z",
    "updated_at": "2024-03-13T17:25:24.347Z",
    "minimum_latency": 60,
    "maximum_latency": 150,
    "upscale_quantity": 1,
    "downscale_quantity": 1,
    "upscale_sensitivity": 1,
    "downscale_sensitivity": 2,
    "upscale_timeout": 0,
    "downscale_timeout": 0,
    "upscale_on_initial_job": true,
    "issue": null
  }
}
```

A description of the new option/feature:

<img width="835" alt="snap-2024-03-13-13 23 31" src="https://github.com/carwow/terraform-provider-hirefire/assets/48658/a6ec4ae3-6e7a-402a-9d7c-bbe34c5e8543">

**Comment by PChambino** [comment]
Hi @stevenharman! Thank you for raising the PRs. I will do a quick test over next week and then release a new version.

**Comment by PChambino** [comment]
I have released v0.7.0-pre. Can you try it and confirm it works as expected?

**Comment by stevenharman** [comment]
🙏 Thank you for the quick turnaround, and for fixing my terrible Golang. 😆

I just confirmed that `v0.7.0-pre` works as expected for these new attributes.

**Comment by PChambino** [comment]
Thanks! No issues on Carwow's terraform project as well, so released v0.7.0. 🚀
